### ISSUE TITLE
Update request: rio 0.1.1 → 0.1.6

### ISSUE THREAD

**Issue by zh4ngx** [NixOS/nixpkgs#330764]
- Package name: rio
- Latest released version: 0.1.6

- Current version on the unstable channel: 0.1.1
- Current version on the stable/release channel: 0.0.39

- [x] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

**Notify maintainers**

@TornaxO7 @otavio @oluceps 
-----

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by cmltaWt0** [comment]
Just a note that the version in unstable channel is broken for Darwin.
https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/applications/terminal-emulators/rio/default.nix#L133

**Comment by zh4ngx** [comment]
Indeed, broken on NixOS / Wayland in general as well, so this update would be greatly appreciated

**Comment by TornaxO7** [comment]
```
       > error: rustc 1.79.0 is not supported by the following package:
       >   rioterm@0.1.3 requires rustc 1.80.0
       >
```
so we have to wait until `rustc 1.80.0` is in `nixpkgs`.

**Comment by TornaxO7** [comment]
@zh4ngx but you can use the flake from the rio repository

**Comment by zh4ngx** [comment]
> ```
>        > error: rustc 1.79.0 is not supported by the following package:
>        >   rioterm@0.1.3 requires rustc 1.80.0
>        >
> ```
> 
> so we have to wait until `rustc 1.80.0` is in `nixpkgs`.

Darn ok

**Comment by TornaxO7** [comment]
I also think that the ryantm bot will automatically create a PR if it's possible to upgrade `rio`.

**Comment by zh4ngx** [comment]
Updated to `0.1.4`

**Comment by zh4ngx** [comment]
Updated to `0.1.5`

**Comment by zh4ngx** [comment]
> Updated to `0.1.5`

update to `0.1.6`

**Comment by zh4ngx** [comment]
> ```
>        > error: rustc 1.79.0 is not supported by the following package:
>        >   rioterm@0.1.3 requires rustc 1.80.0
>        >
> ```
> 
> so we have to wait until `rustc 1.80.0` is in `nixpkgs`.

looks like it just happened. :tada:

**Comment by nyabinary** [comment]
@otavio @TornaxO7 Rust 1.80 in nixpkgs now :P

**Comment by zh4ngx** [comment]
Thanks! Did one of the checks get stuck?

**Comment by TornaxO7** [comment]
> Thanks! Did one of the checks get stuck?

No, we just need to wait for a person with write access to this repo to accept the PR. This could take up some days.

**Comment by zh4ngx** [comment]
Closed by #335474 - much appreciated
